uwsgi/plan.sh

```diff
@@ -0,0 +1,43 @@
+pkg_name=uwsgi
+pkg_description="uWSGI application server container"
+pkg_upstream_url=https://github.com/unbit/uwsgi
+# $HAB_ORIGIN overrides pkg_origin.
+pkg_origin=origin
+pkg_version=2.0.14
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-2.0")
+pkg_source=https://github.com/unbit/${pkg_name}/archive/${pkg_version}.tar.gz
+pkg_shasum=0761056d465061568d97726f985e0a9c236ef8682dd012b60746bc453293b28e
+pkg_dirname=${pkg_name}-${pkg_version}
+pkg_deps=(core/gcc-libs core/glibc core/pcre core/python)
+pkg_build_deps=(core/gcc core/gcc-libs core/make core/pcre core/python)
```

Comment:
I tried to brute force the addition of libgcc_s.so and libpcre.so but yet to no avail.